configuration-api/main/advancedAuth.js

```diff
@@ -22,18 +22,23 @@ const typeDefs = gql`
     secondaryColor: String!
   }
   type UserPreferencies {
-    usrName: String!
+    userName: String!
     language: String!
   }
   type Query {
     listTenants(tenantNames: [String]!): [TenantConfiguration]
-    getUserPreferences(usrName: String!): [UserPreferencies]
+    getUserPreferences(userName: String!): [UserPreferencies]
   }
   type Mutation {
-    modifyUserPreferences(usrName: String!, language: String!): [UserPreferencies]
-    publishTenants(name: String!, icon: String!, primaryColor: String!, secondaryColor: String!): [TenantConfiguration]
-    removeTenants(tenantNames: [String]!): Boolean!
-    modifyTenants(name: String!, icon: String!, primaryColor: String!, secondaryColor: String!): [TenantConfiguration]
+    modifyUserPreferences(userName: String!, language: String!): [UserPreferencies]
+    getTenantConfig(name: String!, icon: String!, primaryColor: String!, secondaryColor: String!): [TenantConfiguration]
+    removeTenantsConfig(tenantNames: [String]!): Boolean!
+    modifyTenantsConfig(
```

Comment:
is this changing multiple tenants' config or a single one? in the second case, let's remove the 's'